packages/katello/katello/katello.spec

```diff
@@ -152,7 +152,6 @@ Requires: qpid-tools
 Requires: /bin/ps
 Requires: rh-mongodb34
 Requires: %{name}-service
-Requires: /usr/bin/psql
```

Comment:
The foreman-debug itself uses psql and thus should handle this. Further, this ties to a specific binary that can change based on Postgres via SCL or not.

Reply:
Given https://community.theforeman.org/t/rfc-deprecate-foreman-debug-in-favor-of-sosreport/12465 should we drop it altogether?

Reply:
We probably should, but is that RFC fully implemented? My read on it is no.

Reply:
sosreport has implemented all the functionality but that version has only landed in EL7. Not Debian derivatives. That doesn't really matter for Katello. Also not sure if katello users really use the upload functionality to our servers.